From 00715f72a3ad59f74889b98c2ad5ccb9b02bcd0c Mon Sep 17 00:00:00 2001
From: jdeolive <Joao.DeOliveiraSilva@windriver.com>
Date: Mon, 5 Feb 2024 14:48:56 -0300
Subject: [PATCH] Removed cephobjectstore.yaml from templates folder.

This patch removes objectstore from the templates thus preventing its 
instalation. Its unnecessary.

Signed-off-by: jdeolive <Joao.DeOliveiraSilva@windriver.com>
---
 .../templates/cephobjectstore.yaml            | 23 -------------------
 1 file changed, 23 deletions(-)
 delete mode 100644 cluster/charts/rook-ceph-cluster/templates/cephobjectstore.yaml

diff --git a/cluster/charts/rook-ceph-cluster/templates/cephobjectstore.yaml b/cluster/charts/rook-ceph-cluster/templates/cephobjectstore.yaml
deleted file mode 100644
index 21177f32b..000000000
--- a/cluster/charts/rook-ceph-cluster/templates/cephobjectstore.yaml
+++ /dev/null
@@ -1,23 +0,0 @@
-{{- $root := . -}}
-{{- range $objectstore := .Values.cephObjectStores -}}
----
-apiVersion: ceph.rook.io/v1
-kind: CephObjectStore
-metadata:
-  name: {{ $objectstore.name }}
-spec:
-{{ toYaml $objectstore.spec | indent 2 }}
----
-{{- if default false $objectstore.storageClass.enabled }}
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: {{ $objectstore.storageClass.name }}
-provisioner: {{ $root.Release.Namespace }}.ceph.rook.io/bucket
-reclaimPolicy: {{ default "Delete" $objectstore.storageClass.reclaimPolicy }}
-parameters:
-  objectStoreName: {{ $objectstore.name }}
-  objectStoreNamespace: {{ $root.Release.Namespace }}
-{{ toYaml $objectstore.storageClass.parameters | indent 2 }}
-{{ end }}
-{{ end }}
-- 
2.25.1

